ha/alert/const.py

```diff
@@ -18,6 +18,10 @@
 
 from ha.const import _DELIM
 
+GET_ONLINE_NODES_CMD = "crm_mon --as-xml"
+GET_LOCAL_NODE_ID_CMD = "crm_node -i"
+GET_LOCAL_NODE_NAME_CMD = "crm_node -n"
+
```

Comment:
GET_ONLINE_NODES_CMD = "crm_mon --as-xml"
GET_LOCAL_NODE_ID_CMD = "crm_node -i"
GET_LOCAL_NODE_NAME_CMD = "crm_node -n"

move them to
ha.const as generic api

Reply:
Done